bin/dep

```diff
@@ -23,23 +23,30 @@ elif [ "$(uname -o)" = "Msys" ]; then
   exe=.exe
 fi
 
-depbin=.dep${exe}
+depbin="${rootdir}/.dep${exe}"
 depurl="${dep_base_url}dep-${os}-amd64${exe}"
 
+if [ -f "$depbin" ]; then
+  version=$($depbin version | grep "^ version" | awk '{print $3}' | awk -F '-' '{print $1}')
+  if [ "$version" != "$depversion" ]; then
+    rm "$depbin"
+  fi
+fi
```

Comment:
I think you could probably do this more easily--without relying on the output formatting of dep, for instance--by storing the $depversion in the $depbin (i think we do this for protoc in the api repo, for instance).  I.e., if the version is encoded in the file name, there's no reason to discover what version the local file is.

Reply:
Ahh, that's a great point -- thanks. Will do.

Reply:
Took that approach in 54d9e788.